lib/routes/requests.js

```diff
@@ -0,0 +1,97 @@
+var Boom = require('boom')
+
+/* NOTES:
+  * What Auth do we use against CouchDB to get/set user doc? (user not logged in)
+  * What url do we send to the user? (how do configure url)
+  * Could we use promises? I think it will clean the code up nicely
+*/
+module.exports = function (server, options, next) {
+  var couchUrl = options.couchdb || options.adapter.location
+  var prefix = options.prefix || ''
+
+  var request = require('request').defaults({
+    json: true,
+    baseUrl: couchUrl,
+    timeout: 10000 // 10 seconds
+  })
+
+  function createPasswordReset (doc, cb) {
+    request.get({
+      url: '/uuids'
+    }, function (err, resp, body) {
+      if (err) {
+        return cb(err)
+      }
+      doc.reset_time_stamp = new Date()
+      doc.reset_token = body.uuids[0]
```

Comment:
just a minor (feel free to ignore we can fix this later): we use camelCase for variables, properties and methods